modules/cluster-logging-collector-log-forward-gcp.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: PROCEDURE
+[id="cluster-logging-collector-log-forward-gcp_{context}"]
+= Forwarding logs to Google Cloud Platform
+
+You can forward logs to Google Cloud Logging in addition to, or instead of, the internal default {product-title} log store.
+
+[NOTE]
+====
+Using this feature with fluentd is not supported.
+====
+
+.Prerequisites
+* {logging-title-uc} Operator 5.5.1 and later
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  outputs:
+    - name: gcp-1
+      type: googleCloudLogging
+      secret:
+        name: gcp-secret
+      googleCloudLogging: <1>
+        projectId : "openshift-gce-devel"
+        logId : "app-gcp"
```

Comment:
```suggestion
        logId : "app-gcp" <2>
```

Reply:
logId, projectId etc. shoud be "logID", "projectID" - if we've released the code that way already I guess we have to live with it.

Reply:
https://github.com/openshift/cluster-logging-operator/pull/1614/files#diff-8f9105693decf749520b1f29a67276eef07601e8873521c61d38a5b93e667ec6